webrtc.html

```diff
@@ -1392,6 +1400,14 @@
                     <p>Otherwise, if <var>description</var> is set as a remote
                     description, then run one of the following steps:</p>
                     <ul>
+                      <li>
+                        <p>If <var>description</var> is set as a remote description,
```

Comment:
This should also be in step 2.2 where it covers _"If the process to apply description fails for any reason..."_.